### ISSUE TITLE
Akka warning for 2.7.0-RC3

### ISSUE THREAD

**Issue by eximius313** [playframework/playframework#8812]
### Play Version (2.5.x / etc)
2.7.0-RC3

### API (Scala / Java / Neither / Both)
Java

### Operating System (Ubuntu 15.10 / MacOS 10.10 / Windows 10)
Windows 7

### JDK (Oracle 1.8.0_72, OpenJDK 1.8.x, Azul Zing)
JDK 9.0.4

### Library Dependencies
Akka?

### Expected Behavior
No warning

### Actual Behavior
```
[WARN] [11/17/2018 00:41:29.223] [main] [ManifestInfo(akka://sbt-web)] Detected possible incompatible versions on the classpath. Please note that a given Akka version MUST be the same across all modules of Akka that you are using, e.g. if you use [2.5.17] all other modules that are released together MUST be of the same version. Make sure you're using a compatible set of libraries.Possibly conflicting versions [2.5.4, 2.5.17] in libraries [akka-persistence:2.5.4, akka-protobuf:2.5.4, akka-actor:2.5.17, akka-remote:2.5.4, akka-cluster:2.5.4, akka-stream:2.5.4, akka-cluster-tools:2.5.4]
```

### Reproducible Test Case
just run `sbt`

**Comment by marcospereira** [comment]
The problem is that we depend on sbt-js-engine 1.2.2 that depends on js-engine 1.2.3 that depends on Akka 2.5.4. So we need to update sbt-web plugins, do new releases, and update them in Play. But this could be annoying when we update for new versions of Akka, so maybe we should instead reconfigure the log level for Akka to avoid this message.

We can also exclude/override the dependencies that are "conflicting".

**Comment by marcospereira** [comment]
> What's the other source of Akka version, in the build classpath (not to be confused with the application classpath of Play)?

I don't think that there are other sources of Akka version out-of-the-box right now. Of course, this can change if some of the dependencies add it. Right now, for RC8, there is no warning anymore.

> I don't think we need to releases new versions of everything every time there's a new Akka release (what a pain! only if we have all our releases automated and streamlined).

I'm against it too.

> But we do need to upgrade all the components of Akka, by redefining them with an upgrade version.

What you mean here? In Play itself or for Play applications? The problem reported here was that there was an sbt plugin bringing Akka artifacts that aren't used by Play and so they weren't evicted.

> (albeit, in the build space, rather than the production space).

And of course, it can happen in production space also. I agree that silencing the warning is the wrong thing to do. It is there exactly to avoid mixing versions.

**Comment by dwijnand** [comment]
> > But we do need to upgrade all the components of Akka, by redefining them with an upgrade version.
> 
> What you mean here? In Play itself or for Play applications? The problem reported here was that there was an sbt plugin bringing Akka artifacts that aren't used by Play and so they weren't evicted.

Ah, I see.  I thought it was in Play itself.

**Comment by marcospereira** [comment]
This is solved in 2.7.0-RC8.
